src/cli/nexus-prisma.ts

```diff
@@ -27,7 +27,7 @@ process.env.DEBUG_HIDE_DATE = 'true'
 generatorHandler({
   onManifest() {
     return {
-      defaultOutput: Path.join(__dirname, '../runtime'),
+      defaultOutput: Path.join(__dirname, '../../../../node_modules/.nexus-prisma'),
```

Comment:
Yeah that's what I'm saying – this looks pretty fragile. Could we use something like `require.resolve()` to  find the module rather than traversing the tree?